tests/unit_tests/test_cli.py

```diff
@@ -25,7 +25,7 @@
 
 @pytest.fixture
 def docker(request, mock_server, mocker, monkeypatch):
-    wandb_args = {"check_output": b'["wandb/deepo@sha256:abc123"]'}
+    wandb_args = {"check_output": b"wandb/deepo@sha256:abc123"}
```

Comment:
@vanpelt why was that change needed? it breaks things locally for me; i can point exactly where things break in pydockercreds lib.

Reply:
rolling it back now to see if it makes the ci happy, here's a screenshot
<img width="1342" alt="image" src="https://user-images.githubusercontent.com/7557205/176779010-8f1eb160-acfe-45e5-9c4c-19f14c05a40f.png">